src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -495,26 +525,27 @@ def _DefaultProtectionPoints(name,
   This creates a default set of points by regular sampling of the contour and gridding
   the interior of the DPA zone. The contour and zone are separated in front and back,
   using the US border (with a buffer) as the delimitation.
+  If the DPA is a Point, a single point is returned however.
 
   Args:
-    num_pts_front_border: number of points in the front border
-    num_pts_back_border: number of points in the back border
-    num_pts_front_zone: number of points in the front zone
-    num_pts_back_zone: number of points in the back zone
-    front_us_border_buffer_km: buffering of US border for delimiting front/back.
+    dpa_geometry: The DPA geometry as a |shapely.Polygon or Point|.
+    num_pts_front_border: Number of points in the front border.
+    num_pts_back_border: Number of points in the back border.
```

Comment:
It might be helpful to note that "num_pts_front_border" and "num_pts_back_border" >2. I actually tried 1 and got an error message "ZeroDivisionError: float division by zero" on line 553, in SampleLine(). :)

Reply:
Ok I will put a check to handle this case.